beaker_kernel/lib/subkernels/python.py

```diff
@@ -129,3 +181,48 @@ async def setup(self):
 del importlib, os, site, sys
 """
         await self.context.execute(setup_code)
+
+    def format_kernel_state(self, state):
+        formatted_state = {
+            "modules": {},
+            "variables": {},
+            "functions": {}
```

Comment:
I recommend adding `classes` here. Currently classes are showing up under functions, which is confusing.